v3/glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+* _channel_ : **TODO: find good description for channel**
+* _cloaking_ : method used to hides telehash traffic on the wire by randomizing all data sent
+* _Cipher Set (CS)_ : a collection of crypto algorithms with a given _cipher set id (CSID)_
+* _Cipher Set ID (CSID)_ : predefined hex number identifying a _cipher set (CS)_
+* _endpoint_ : a participant in the telehash network identified by a **(TODO: or multiple?)** _hashname_
```

Comment:
1:1, an "endpoint" is the entity identified by a "hashname" (an app may create multiple hashnames but as far as the protocol is concerned they are distinct endpoints, coupled only by whatever the app might do)